### ISSUE TITLE
[TF2] [Feature Request] Alt. Game Modes should be checked by default

### ISSUE THREAD

**Issue by agrastiOs** [ValveSoftware/Source-1-Games#2056]
If a player launches up Casual for the first time, only Core Game Modes will be checked. And that shows. Alt. Game Modes health is horrible. Most are gray, and some are brown at most. If we rechecked the Alt. Game Modes for everyone,  we should expect boosts to the population of Alt. Game Modes.

**Comment by DrCactusTF2** [comment]
Yes, this is really neccessary, especially because the Jungle Inferno Campaign has alternative map objectives, which are almost impossible to do due to the low populations.
